packages/x-tailwindcss/src/x-tailwind-plugin/components/picture/default.ts

```diff
@@ -9,7 +9,6 @@ import { TailwindHelpers } from '../../../types';
 export function pictureDefault(helpers: TailwindHelpers) {
   const { theme } = helpers;
   return {
-    display: 'block',
```

Comment:
Removed as this class ends up being applied on a `div`, & it has `display: block` applied by default in browsers.